src/redfishpower/redfishpower.c

```diff
@@ -338,20 +395,23 @@ static void stat_cmd(CURLM *mh, char **av)
         }
         hostsptr = &lhosts;
     }
-    else
-        hostsptr = &hosts;
+    else {
+        /* if plugs are used, use all plugs, otherwise use all hosts */
+        if (hostlist_count(plugs) > 0)
+            hostsptr = &plugs;
+        else
+            hostsptr = &hosts;
+    }
```

Comment:
This function also uses hosts where plugs would be more appropriate (e.g. hostsptr) and the comment feels slightly off since plugs are always used - the question is whether they are defined by `setplugs` or not